src/CMakeLists.txt

```diff
@@ -245,7 +245,12 @@ if(CCACHE AND NOT CMAKE_CXX_COMPILER_LAUNCHER AND NOT CMAKE_C_COMPILER_LAUNCHER)
 endif()
 
 # Python
-find_package(PythonInterp)
+# https://cmake.org/cmake/help/latest/policy/CMP0148.html
+if(${CMAKE_VERSION} VERSION_LESS_EQUAL "3.12")
+  find_package(PythonInterp)
+else ()
+  find_package(Python COMPONENTS Interpreter)
+endif ()
```

Comment:
This whole block shouldn't be necessary anymore, could you try deleting it?

Reply:
Hi, had another go at this. The CMake commit is dropped. Let's see if the CI is happy.